src/mpid/ch4/netmod/ucx/ucx_recv.h

```diff
@@ -37,7 +37,14 @@ static inline void MPIDI_UCX_recv_cmpl_cb(void *request, ucs_status_t status,
 
     if (unlikely(status == UCS_ERR_CANCELED)) {
         MPIR_STATUS_SET_CANCEL_BIT(rreq->status, TRUE);
-    } else {
+    }
+    else if (unlikely(status == UCS_ERR_MESSAGE_TRUNCATED)) {
```

Comment:
We may get some problems with the "count" , since we can have any value in there - that's why I omited it. However, it also works the other way - so I am OK with changing it.

Alternatively we can just set no filed - according to the UCX specification, the values in INFO are not defined in the case of an error.